src/extractor/guidance/turn_lane_augmentation.cpp

```diff
@@ -283,16 +283,26 @@ LaneDataVector handleNoneValueAtSimpleTurn(LaneDataVector lane_data,
         // a pgerequisite is simple turns. Larger differences should not end up here
         // an additional line at the side is only reasonable if it is targeting public
         // service vehicles. Otherwise, we should not have it
-        BOOST_ASSERT(connection_count + 1 == lane_data.size());
-
-        lane_data = mergeNoneTag(none_index, std::move(lane_data));
+        //
```

Comment:
This is more like a workaround. We should never arrive here, when this is not true. Not handling the none-tag here simply fails out later instead of earlier. Will have to look into it.